api-report/tree.api.md

```diff
@@ -364,6 +370,35 @@ export class FieldKind<TEditor extends FieldEditor<any> = FieldEditor<any>> {
 // @public
 export type FieldKindIdentifier = Brand<string, "tree.FieldKindIdentifier">;
 
+declare namespace FieldKinds {
+    export {
+        lastWriteWinsRebaser,
+        replaceRebaser,
```

Comment:
we don't want to export everything in FieldKinds. I'm not sure how we want to handle this though: I'll look for a better solution.

Reply:
In `packages/dds/tree/src/index.ts` do:

```
// Export subset of FieldKinds in an API-Extractor compatible way:
import { FieldKind, FieldKinds as FieldKindsOriginal } from "./feature-libraries";
interface FieldKinds {
    value: FieldKind;
    optional: FieldKind;
    sequence: FieldKind;
}
const FieldKinds: FieldKinds = FieldKindsOriginal;
export { FieldKinds };
```

I have confirmed this works, and avoids the unwanted exports.

Reply:
done in most recent commit aeb65

Reply:
You removed the comment when you did this. I think it would be better to keep the comment.

Reply:
Done in recent commit 33aba8